tests/integration/cattletest/core/test_link.py

```diff
@@ -2,23 +2,24 @@
 
 
 @pytest.fixture(scope='module')
-def link_network(admin_client, sim_context):
-    nsp = create_agent_instance_nsp(admin_client, sim_context)
-    create_and_activate(admin_client, 'linkService',
+def link_network(internal_test_client, sim_context):
+    nsp = create_agent_instance_nsp(internal_test_client, sim_context)
+    create_and_activate(internal_test_client, 'linkService',
                         networkServiceProviderId=nsp.id,
                         networkId=nsp.networkId)
 
-    return admin_client.by_id_network(nsp.networkId)
+    return internal_test_client.by_id_network(nsp.networkId)
 
 
-def test_link_instance_stop_start(admin_client, sim_context, link_network):
-    target1 = create_sim_container(admin_client, sim_context,
+def test_link_instance_stop_start(internal_test_client, sim_context,
```

Comment:
For this entire module (test_link.py) it looks like we've defaulted to using the internal_test_client. But admin_client should be able to do most instanceLink related things.

Seems to me that we should try to use admin_client as much as possible here.

Am I missing something @ibuildthecloud ?